man/man7/hashbang.7

```diff
@@ -38,9 +38,6 @@ code readability.
 
 ruby - A dynamic, object-oriented general-purpose programming language.
 
-haskell - A standardized, general-purpose programming language with non-strict
```

Comment:
```
mihi@da1:~$ ghc
Could not find the database of available applications, run update-command-not-found as root to fix this
ghc: command not found
```
According to https://packages.debian.org/stable/i386/ghc/filelist, the binary should be called `/usr/bin/ghc`